packages/components/src/menu-actions/Menu.tsx

```diff
@@ -22,12 +58,12 @@ class Menu extends PureComponent {
     this.handleMouseLeave = this.handleMouseLeave.bind(this);
     this.handleCloseMenu = this.handleCloseMenu.bind(this);
 
-    this.container = null;
+    this.container = React.createRef();
     this.oldFocus = document.activeElement;
-    this.rAF = null;
+    this.rAF = 0;
 
     const { options } = props;
-    let keyboardIndex = options.initialKeyboardIndex;
+    let keyboardIndex = options.initialKeyboardIndex || -1;
     if (!Number.isInteger(keyboardIndex)) {
       keyboardIndex = -1;
     }
```

Comment:
Use `??` instead of `||`, otherwise can't select `0`. Also, shouldn't need the next `if` condition, as it will always be a number
```suggestion
    const keyboardIndex = options.initialKeyboardIndex ?? -1;
```

Reply:
Good catch